desktop/fontforge.appdata.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2014 Joe Da Silva <digital@joescat.com> -->
 <component type="desktop">
-  <id>fontforge</id>
-  <launchable type="desktop-id">fontforge.desktop</launchable>
+  <id>io.github.Fontforge</id>
```

Comment:
I prefer FontForge too, but I think we should follow internet standards (all lowercase).
I still recall visiting some websites that had trouble with uppercase - This was a while back in the past, but in any case, safest to aim for lowest common denominator.
A glance at /usr/share/appdata seems to be mostly lowercase except for a few subsets.

I'm ok to merge this as-is. I would like to follow-up with a desktop directory upgrade.

Reply:
From the spec, it's `{tld}.{vendor}.{product}`, and the example provided is `com.hugski.ColorHug2`.

Although following that spec, it's kinda broken anyway, since we no longer use fontforge.org as the domain, and `github` isn't really the vendor...

Reply:
Thought so too. Which was why I think staying with fontforge.desktop would be better. The upstream flatpak probably should reconsider going with the older desktop spec.